Cargo.toml

```diff
@@ -16,14 +16,11 @@ name = "http-service"
 version = "0.4.0"
 
 [workspace]
-members = ["http-service-hyper", "http-service-lambda", "http-service-mock"]
+members = ["http-service-hyper", "http-service-mock", "http-service-h1"]
```

Comment:
Should we keep the lambda backend?

Reply:
There is no way to make it work at the moment, PRs are not getting merged and it is still on http@0.1 making it incomaptible with the new version. So I am not seeing the reason for keeping dead code around.